core/templates/dev/head/services/PromoBarService.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service Promo bar.
+ */
+
+oppia.factory('PromoBarService', [
+  '$http', 'ENABLE_PROMO_BAR',
+  function($http, ENABLE_PROMO_BAR) {
+    return {
+      getPromoBarData: function() {
+        return $http.get('/promo_bar_handler', {}).then(
+          function(response) {
+            if (ENABLE_PROMO_BAR) {
+              return {
+                promoBarEnabled: response.data.promo_bar_enabled,
+                promoBarMessage: response.data.promo_bar_message
+              };
+            } else {
```

Comment:
You don't need else here, just return with default values is enough.

Reply:
Done.